src/devices/SoftPwm/SoftwarePwmChannel.cs

```diff
@@ -176,5 +177,14 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        /// <inheritdoc />
+        public override ComponentInformation QueryComponentInformation()
+        {
+            var self = new ComponentInformation(this, "Software Pwm driver");
+            self.AddSubComponent(_controller.QueryComponentInformation());
+            self.Properties["Pin"] = _pin.ToString(CultureInfo.InvariantCulture);
```

Comment:
I suggest we should change pin to always produce invariant culture information on regular ToString. I don't know how other programmers feel about translating diagnostic messages but I'm personally always annoyed by wacky translation done in my native tongue. It might make sense for other things but for diagnostics it just doesn't feel right